pkg/controller/nodes.go

```diff
@@ -47,6 +47,11 @@ const (
 	tunnelIDIncr = 2
 )
 
+// Name of the taint to add to nodes that are not ready
+const (
+	ACIContainersTaintName string = "aci-containers-host/unavailable"
+)
+
```

Comment:
@smshareef  How are we making sure if we are removing taint only added by controller?
With this fix key-name we will remove taint not set by controller right? Can we generate taint-key(key=value:effect) dynamically and set it??

Reply:
@PariyaAshok Yes, if the taint with this key and effect NoSchedule is added, then we will remove. We can document that the taint with this key is managed by the ACI-CNI.

If we generate the taint key dynamically, then the same needs to be communicated between the controller and host-agent.